README.md

```diff
@@ -198,6 +204,43 @@ const player = new Player({
 });
 ```
 
+### Setting up a playback configurations
+
+If you need more option to playback configuration you can add new parameter `playbackConfig`
+
+#### Configuring through code
```

Comment:
You can remove this sub-section since it is the only one present under the main headline.

```suggestion
```